.github/actions/environment/action.yml

```diff
@@ -5,6 +5,7 @@ runs:
   steps:
     - run: |
         echo VARDIR=/tmp/tnt | tee -a $GITHUB_ENV
+        echo SERVER_START_TIMEOUT=290 | tee -a $GITHUB_ENV
```

Comment:
Please fix typo in the commit message. It should be 290.

Reply:
Done.